base/docker-compose.yml

```diff
@@ -32,3 +32,5 @@ services:
         PIGCS2_VERSION: ${PIGCS2_VERSION}
         PMAC_VERSION: ${PMAC_VERSION}
         OPCUA_VERSION: ${OPCUA_VERSION}
+        SCALER_VERSION: ${SCALER_VERSION}
```

Comment:
This is in the wrong commit. It should be in ef8d83cd30f752ea8accffe7adfae99f8efe32bd.